frontend/src/App.jsx

```diff
@@ -50,6 +50,8 @@ import "assets/css/theme.scss"
 import "./App.scss"
 import "assets/css/header.scss"
 
+const ELEMENT_LIST_BUFFER_TIMEOUT_MS = 10
```

Comment:
Add comment explaining what this is. Something like: "We update the elements list in the state in batches. This is how often we apply those batches."